app/views/public_pages/faq.html.erb

```diff
@@ -0,0 +1,131 @@
+<% content_for :page_title, "FAQ" %>
+<% content_for :body_class, "faq" %>
+
+<main role="main">
+  <section class="slab slab--white">
+    <div class="grid">
+      <div class="grid__item width-three-fourths">
+        <h1 class="h2">
+          Frequently asked questions
+        </h1>
+        <p>
+          Click below for responses to frequently asked questions. Chat with us if you still have questions!
+        </p>
+
+        <h2 class="h3 spacing-above-35">
+          Help getting started
+        </h2>
+
+        <div class="accordion">
+          <button href="#" class="accordion__button" aria-expanded="true" aria-controls="a1">
+            What should I do if I haven’t filed in several years?
+          </button>
```

Comment:
Nonblocking, but if these were `h3` tags wrapped in `a` tags, it would improve keyboard navigation and thereby any assistive tech navigation, including screen readers.

Reply:
ah, i was going by the honeycrisp examples, assuming given the aria attributes that it had already been optimized for accessibility. 

it's easy enough to change this with just a small piece of additional css for the accordion molecule, so I'm submitting an update to do that. I tested with the WAVE plugin before and after this change, and while I must admit I'm not totally sure what I should be looking for, I observed:
1) no errors with the button approach or the a>h3 approach
2) able to use tab to navigate and use the enter key to open the FAQ question with both approaches too
3) the a>h3 approach introduces an orange border highlight on the h3s when you navigate which I assume is a desired improvement for accessibility. however, it also introduces an extra tab press to navigate from question to question, and the space key, which also opened questions with the button approach, no longer works. i'm not sure if those things are problems or just..different

i'm going to go ahead and submit this update on the assumption that the orange border thingy is a plus and the other changes I observed are not a big deal.